src/StackAdmin/Azs.Fabric.Admin/Tests/src/Volume.tests.ps1

```diff
@@ -68,11 +75,15 @@ InModuleScope Azs.Fabric.Admin {
                 $Volume.Name     | Should Not Be $null
                 $Volume.Type     | Should Not Be $null
 
-                # Storage Pool
-                $Volume.FileSystem       | Should Not Be $null
-                $Volume.RemainingSizeGB  | Should Not Be $null
-                $Volume.SizeGB           | Should Not Be $null
-                $Volume.VolumeLabel      | Should Not Be $null
+                # Volume
+                $Volume.TotalCapacityGB      | Should Not Be $null
+                $Volume.RemainingCapacityGB  | Should Not Be $null
+                $Volume.HealthStatus         | Should Not Be $null
```

Comment:
Are any of these an ENUM?  If so you should validate they are valid values

Reply:
No, they are string.